### PR TITLE
PUB-1914 - Readme Changes

### PR DESCRIPTION
**Pull request by furnivall**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/PUB-1914

### Change description ###
Refreshed readme

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,304 +1,275 @@
...
 
-    Provides code coverage metrics for Java code via integration with JaCoCo.
-    You can create the report by running the following command:
+##### Local development
 
-    ```bash
-      ./gradlew jacocoTestReport
-    ```
+- [Azurite](https://learn.microsoft.com/en-us/azure/storage/common/storage-use-azurite) - Local Azure emulator used along with Azure Storage explorer for local storage.
+- [Azure Storage Explorer](https://azure.microsoft.com/en-us/products/storage/storage-explorer) - Used for viewing and storing blobs within an Azurite instance locally.
```
Comment (joshblackmoor) [inline README.md:85]: I think we should add Postgres in here too

File: README.md [hunk 2]
```diff
@@ -1,304 +1,275 @@
...
- * [Separate, per-dependency thread pools](https://github.com/Netflix/Hystrix/wiki/How-it-Works#isolation)
- * [Semaphores](https://github.com/Netflix/Hystrix/wiki/How-it-Works#semaphores), which you can use to limit
- the number of concurrent calls to any given dependency
- * [Request caching](https://github.com/Netflix/Hystrix/wiki/How-it-Works#request-caching), allowing
- different code paths to execute Hystrix Commands without worrying about duplicating work
+If you wish to test a flyway script locally, you will first need to clear the `flyway_schema_history` table then set the environment variable `ENABLE_FLYWAY` to true.
 
-## License
+## Monitoring and Logging
+We utilise [Azure Application Insights](https://learn.microsoft.com/en-us/azure/azure-monitor/app/app-insights-overview) to store our logs. Ask a teammate for the specific resource in Azure to access these.
```
Comment (joshblackmoor) [inline README.md:244]: Maybe add in something about Dynatrace as that also covers the monitoring side?

**Review by joshblackmoor — COMMENTED** [review]
looks good, added few suggestions

File: README.md [hunk 3]
```diff
@@ -1,304 +1,275 @@
...
-The template is a working application with a minimal setup. It contains:
- * common plugins and libraries
- * docker setup
- * swagger configuration for api documentation ([see how to publish your api documentation to shared repository](https://github.com/hmcts/reform-api-docs#publish-swagger-docs))
- * code quality tools already set up.
- * Hystrix circuit breaker enabled.
- * MIT license and contribution information.
- * Helm chart using chart-java.
+## Overview
+`pip-data-management` is a microservice that deals with most operations relating to data persistence within the Court and Tribunals Hearing Service (CaTH hereafter) written with Spring Boot/Java.
```
[the remark this answers is not in the document]
Comment (furnivall) [reply README.md]: resolved

File: README.md (hunk 1 above)
Comment (furnivall) [reply README.md:85]: resolved

File: README.md (hunk 2 above)
Comment (furnivall) [reply README.md:246]: resolved

**Review by joshblackmoor — APPROVED** [review]

**Review by ChrisS1512 — CHANGES_REQUESTED** [review]

**Pull request merged**